Tests/Suites/Extensions/NSDictionary+TogglesTests.swift

```diff
@@ -0,0 +1,51 @@
+//  NSDictionary+TogglesTests.swift
+
+import XCTest
+@testable import Toggles
+
+final class NSDictionary_TogglesTests: XCTestCase {
+    
+    func testEmptyNSDictionaryConversion() {
+        let emptyNSDictionary: NSDictionary = [:]
+        
+        XCTAssertThrowsError(try emptyNSDictionary.convertToToggleDataSource()) { error in
+            XCTAssertEqual(error as? NSDictionary.ToggleConversionError, .nsDictionaryEmpty)
+        }
+    }
+    
+    func testInvalidFormatNSDictionaryConversion() {
```

Comment:
Do we think this test tests something more compared to the one below?
I would just keep the second one.

Reply:
Well it was a case where the first value is valid but then another one isn't.

My code (before submitting) failed this test initially so I thought it was worth keeping in.. happy to remove if you dont think its necessary now though